CHANGELOG.md

```diff
@@ -20,6 +20,19 @@ _None_
 
 _None_
 
+## 1.3.0
+
+### New Features
+
+* Support for a `version.properties` to manage app versioning - all existing paths remain intact and new paths are only used when a `version.properties` file is present.
+* Add support for providing an `app:` parameter to most versioning-related actions to allow support for multiple apps hosted in a monorepo
+* Supporting the new `version.properties` file also allows for the `HAS_ALPHA_VERSION` variable to be removed as the alpha reference in the properties file will be used going forward.
+* Clients adopting the new `version.properties` will need to implement a gradle task named `updateVersionProperties` to update the `version.properties` file.
+
+### Bug Fixes
+
+* Some cleanup to how we scope variables in some of our actions
```

Comment:
I just realized that this item  should, by definition, have been in "internal changes" not "bug fixes" (since it doesn't fix any bug we had but instead is only an internal refactoring with no impact on users of the plugin)

Reply:
Yeah this makes sense, I'll change the heading here to Internal Changes

Reply:
Updated in https://github.com/wordpress-mobile/release-toolkit/pull/280/commits/16a2d27fe0620b704b4211bc17a209ab70c0fd8f

Reply:
GitHub doesn't mark this conversation as outdated, but it is as one can see by looking at the content and the commits history.

I'm going to mark it as resolved.

Reply:
@mokagio Thanks. FYI, GitHub didn't auto-flag this as outdated because my comment was on line 34 only, but what was changed to address this was on line 32, so GitHub couldn't realize it was related or guess that the change made on line 32 would outdate the comment on line 34 😉